prow/config/jobs/tools-1.12.yaml

```diff
@@ -81,13 +81,13 @@ jobs:
   - perf_dashboard
   - deploy
   name: deploy-perf-dashboard
+  service_account_name: prowjob-default-sa
```

Comment:
One thing I am a bit confused about is the naming here. "default sa" seems to suggest that If I don't set any SA name I would get it - but in this case its actually giving extremely high privileges? 

Am I misunderstanding something or is the name just causing confusion?

Reply:
`istio-prow-test-job@istio-testing.iam.gserviceaccount.com` has way more privileges than `prow-deployer@istio-testing.iam.gserviceaccount.com`, and I didn't know who set that up..

Also it seems we are already using it for all Prow jobs - https://github.com/istio/test-infra/blob/master/prow/config.yaml#L25...

I can make two changes:
1. Remove https://github.com/istio/test-infra/blob/master/prow/config.yaml#L25
2. Change the name in https://github.com/istio/test-infra/blob/master/prow/cluster/build/prowjob_service_accounts.yaml#L10 to `prowjob-admin-sa`, and also update the names in my PR.

Does the change look right to you?

Reply:
Updated. Please take a look at the latest commit for delta changes.